### ISSUE TITLE
Active drawing mark cannot regain focus by clicking

### ISSUE THREAD

**Issue by sktw** [zooniverse/front-end-monorepo#7177]
## Package

- lib-classifier

## Describe the bug

In drawing tasks without sub-tasks, once the active mark loses focus, it cannot be regained by clicking on the mark. 

## To Reproduce

1. Navigate to a suitable drawing task ([link](https://www.zooniverse.org/projects/met-rhonda/irish-weather-rescue/classify/workflow/28447/subject-set/131644/subject/113435303)).
2. Draw a line on the subject.
3. Cause the mark to lose focus, for example by clicking or tabbing away.
4. Click the mark again. Notice that the mark does not have focus - for example, you can't delete the mark by pressing 'backspace'.

## Expected behavior

Clicking a mark should cause it to gain focus.

## Device information

Ubuntu 20.04.6
Chrome 143, Firefox 136

## Additional context

- Note that this bug does not prevent a mark gaining focus by tabbing to it.
- Drawing tasks with sub-tasks are unaffected since clicking the mark opens the sub-task popup rather than focusing the mark (although it gains focus when the popup closes). 

- The issue here is that a mark does not lose its active status when it loses focus, unless focus is lost to another mark. For example, if a second line is drawn in the reproduction steps above, it is possible to transfer focus between the two lines by clicking them.
- This bug appears to be a regression introduced in #3817; the logic to deactivate the mark was (and still is) contained in the `onBlur` handler of the [DeleteButton](https://github.com/zooniverse/front-end-monorepo/blob/main/packages/lib-classifier/src/plugins/drawingTools/components/DeleteButton/DeleteButton.jsx). However, as the button can no longer receive keyboard focus, the handler doesn't run.

**Comment by eatyourgreens** [comment]
I think this might be fixed by #7574. @goplayoutside3 ran into the same bug during her review: clicking on a drawn mark doesn't select it.
